### ISSUE TITLE
how to solve this error?

### ISSUE THREAD

**Issue by DDDivano** [Vladkryvoruchko/PSPNet-Keras-tensorflow#50]
![image](https://user-images.githubusercontent.com/10734244/48388201-7f4e8200-e733-11e8-96f0-f5f5b601c2c5.png)

**Comment by sainttelant** [comment]
@DDDivano , yes, i have the same errors as you, and i am trying to print the type of d, and its values, in order to check what happened there, have you solved it yet?

**Comment by sainttelant** [comment]
@DDDivano  
i found the reason why the error occurred, it was so tricky, d's key() is "image", not "anno"  as we expected, the codes must have bugs i assert................................



 d.values {'image': '0a1b45df-3afb5092.jpg'}
type of D <class 'dict'>
